### DISCUSSION TITLE
Unable to start

### DISCUSSION THREAD

**Discussion by malik6524** [jbsparrow/CyberDropDownloader#372] [General]
Hadn't used the program in about a week and when I try to use the start.bat file I get this error

> from cyberdrop_dl.managers.leaky import LeakyBucket
  File "C:\Users\name\Documents\Cyberdrop\venv\Lib\site-packages\cyberdrop_dl\managers\leaky.py", line 8, in <module>
    from aiolimiter.compat import wait_for
ModuleNotFoundError: No module named 'aiolimiter.compat'

**Comment by NTFSvolume** [comment]
Download the newest bat script from the release page.

**Reply by malik6524** [reply]
> Download the newest bat script from the release page.

Thanks. This worked out.

Also, by any chance, did this change make previous download reset or something? It seems some kemono pages that I have previously downloaded whole have reset and now I have to re-download the whole artists page. I also made sure I didn't have ignore history set to true, it's on the default false. 

And while I have you here, is there a way to set specified dates such as 1/1/2024 to 1/20/2024 from within the settings file for example? Thank you NTF.

**Reply by NTFSvolume** [reply]
1. You have to put the new new start script in the same folder as the old one, to make sure they use the same database. It will replace the old one

2. You can't set a date range to scrape

**Comment by unr** [comment]
Sadly, unable to run the latest pip version due to this `aiolimiter.compat` issue


And when I try to run the macOS .command from the releases, it refuses to use my brew `python3`

Stuck as to getting this to run again. :(

Any ETA on fixing the aiolimiter compat issue?

**Reply by dexter-amico** [reply]
You can try using the the new starting script as explained by NTFS, or you can manually uninstall aiolimiter and reinstall it forcing to version 1.1.0. The second method was that I have done myself.

**Reply by unr** [reply]
Installing explicit version got it working again, thank you.

**Comment by NTFSvolume** [comment]
Proper fix implemented in v6.0.0
